test/templates/test_struggling_outpost-wood_gnome.lua

```diff
@@ -0,0 +1,35 @@
+include "test/scripting/check_game_end.lua"
+
+map = game.map
+
+winner = 4
+p = game.players[winner]
+
+run(function()
+  sleep(1000)
+  sf = p:get_buildings("empire_headquarters")[1].fields[1]
+
+  local ff = {}
+  for d = 1,6 do
+    ff[d] = sf
+  end
+
+  for s = 1, 6 do
```

Comment:
:think: Ah, this goes 6 steps in every direction. A comment would make this more obvious.

Reply:
or maybe
```
for i, dir in ipairs({"ln", "tln", "trn", "rn", "brn", "bln"} do
   f = sf[dir][dir][dir][dir][dir][dir]
   print(_("Placing forester ...
```